appveyor.yml

```diff
@@ -4,23 +4,23 @@ image: Visual Studio 2017
 
 platform: x64
 
-build:
-  verbosity: minimal
-
 branches:
-  except:
-    - /.*nist.*/
-    - /master-new-.*/
+  only:
+    - master

```

Comment:
Does this mean we only get AppVeyor builds once we've merged down to master, but not on pull requests?

Reply:
I'm not sure, I just thought I'd whitelist instead of blacklist. Working on it though ...